example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -0,0 +1,229 @@
+//
+// Example that demonstrates Offboard velocity control using LOCAL_NED & BODY_NED
+//
+// Author: Julian Oes <julian@oes.ch>, Shakthi Prashanth <shakthi.prashanth.m@intel.com>
+#include <iostream>
+#include <cmath>
+#include <thread>
+#include <chrono>
+#include <dronecore/dronecore.h>
+
+using namespace dronecore;
+
+#define ERROR_CONSOLE_TEXT "\033[31m" //Turn text on console red
+#define TELEMETRY_CONSOLE_TEXT "\033[34m" //Turn text on console blue
+#define NORMAL_CONSOLE_TEXT "\033[0m"  //Restore normal console colour
+
+void wait_msec(int msec)
+{
+    std::this_thread::sleep_for(std::chrono::milliseconds(msec));
+}
+
+void wait_sec(int sec)
+{
+    std::this_thread::sleep_for(std::chrono::seconds(sec));
+}
+
+int main(int argc, char **argv)
+{
+    DroneCore dc;
+    std::string offb_mode = (argc > 1) ? argv[1] : "";
+
+    if (argc == 1 || (offb_mode != "local-ned" && offb_mode != "body-ned")) {
+        std::cerr << "Usage: offboard <coordinate-frame>\n" <<
+                  "coordinate-frame: \"local-ned\", or \"body-ned\""
```

Comment:
if we default to local then this would explain that we are using local by default and explain how to enable body ned

Reply:
We need to report usage of the example anyway even if we default to local. So, I thought we shall leave this to user.

Reply:
IMO code should show both modes at work - not an either/or choice determined by user. That way you can show off the nuances of using each approach.

Reply:
@hamishwillee that is fair enough..